File: Day_24/Q96.java
import java.util.*;

class Q96{
    static Scanner sc = new Scanner(System.in);

    //Function To Remove Duplicate Characters
    static String remove_duplicate_char(String str){
        String ans = "";
        HashSet<Character> st = new HashSet<>();

        for(int i = 0;i < str.length();i++){
            char ch = str.charAt(i);

            //Check Duplicate Character
            if(!st.contains(ch)){
                ans += ch;
                st.add(ch);
            }
        }

        return ans;
    }

    public static void main(String args[]){
        String str = sc.nextLine();

        //Print Result
        System.out.println(remove_duplicate_char(str));

        sc.close();
    }
}

File: Day_24/Q96.cpp
#include<bits/stdc++.h>
using namespace std;

//Function To Remove Duplicate Characters
string remove_duplicate_char(string str){
    string ans = "";
    unordered_set<char> st;

    for(int i = 0;i < str.length();i++){

        //Check Duplicate Character
        if(st.find(str[i]) == st.end()){
            ans += str[i];
            st.insert(str[i]);
        }
    }

    return ans;
}

int main(){
    string str;
    getline(cin,str);

    //Print Result
    cout << remove_duplicate_char(str);

    return 0;
}
